lib/cost_report_workflow.rb

```diff
@@ -63,7 +63,7 @@ def dump_solr_records
   end
 
   def split_solr_records
-    system("split -l #{chunk_size} #{allrecords_ndj} #{working_directory}/records_")
+    system("split -d -a 5 --additional-suffix=.ndj -l #{chunk_size} #{allrecords_ndj} #{working_directory}/records_")
```

Comment:
I like these `split` parameters. Didn't know it could do all that!
I will try to remember this for future use.